src/app/models/get-access-file.ts

```diff
@@ -0,0 +1,30 @@
+import { FileFormat, PermanentFile } from './file-vo';
+
+export interface HasFiles {
+  FileVOs: PermanentFile[];
+}
+
+function getArchivematicaAccess(files: PermanentFile[]) {
+  return files.find((file) => file.format === FileFormat.ArchivematicaAccess);
+}
+
+function getPrioritizedConvertedFile(files: PermanentFile[]) {
+  return files
+    .filter((file) => file.format === FileFormat.Converted)
+    .sort((a, b) => {
+      const preferredType = 'pdf';
+      return (
+        Number(b.type.includes(preferredType)) -
+        Number(a.type.includes(preferredType))
```

Comment:
Is this cursed? I can change this to be more readable, but the potential for this sorting function to be a single subtraction was irresistible. I'm using the `Number` constructor here because `+x - +y` seemed even more confusing.

Reply:
I think this is reasonable